library/ecp_invasive.h

```diff
@@ -31,6 +31,20 @@
 
 #if defined(MBEDTLS_TEST_HOOKS) && defined(MBEDTLS_ECP_C)
 
+#if defined(MBEDTLS_ECP_DP_SECP224R1_ENABLED) ||   \
+    defined(MBEDTLS_ECP_DP_SECP256R1_ENABLED) ||   \
+    defined(MBEDTLS_ECP_DP_SECP384R1_ENABLED)
+/* Preconditions:
+ *   - bits is a multiple of 64 or is 224
+ *   - c is -1 or -2
+ *   - 0 <= N < 2^bits
+ *   - N has room for bits+64 bits
+ *
+ * Set N to c * 2^bits + N.
```

Comment:
Minor: I find documentation more readable in the order "description, preconditions" than "preconditions, description".

Reply:
The description is kind of a postcondition, so I find it more readable to have it afterwards.